pkg/cri/server/container_stop.go

```diff
@@ -129,9 +130,13 @@ func (c *criService) stopContainer(ctx context.Context, container containerstore
 		if err != nil {
 			return errors.Wrapf(err, "failed to parse stop signal %q", stopSignal)
 		}
-		log.G(ctx).Infof("Stop container %q with signal %v", id, sig)
-		if err = task.Kill(ctx, sig); err != nil && !errdefs.IsNotFound(err) {
-			return errors.Wrapf(err, "failed to stop container %q", id)
+
+		if container.IsSignaled == nil ||
```

Comment:
For the case where the pointer is nil let's output an error indicating we were unable to ensure the signal was not sent twice...